app/include/user_modules.h

```diff
@@ -59,6 +59,7 @@
 //#define LUA_USE_MODULES_STRUCT
 //#define LUA_USE_MODULES_SWITEC
 //#define LUA_USE_MODULES_TM1829
+#define LUA_USE_MODULES_TLS
```

Comment:
I think we should keep modules-list file as clean as possible. Maybe (as an another way) remove `#if defined` line, it may get code cleaner, but adds some points in dependency matrix.